<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>633</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioButton_mode1_test">
         <property name="toolTip">
          <string>逐个测试密码本中的密码，直到检索到正确的密码</string>
         </property>
         <property name="text">
          <string>测试密码</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_mode1_extract">
         <property name="toolTip">
          <string>测试密码并解压到本地</string>
         </property>
         <property name="text">
          <string>解压文件</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_try_unknown_filetype">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;忽略文件类型，对所有类型的文件进行测试或解压&lt;/p&gt;&lt;p&gt;注意：&lt;/p&gt;&lt;p&gt;1. 启用该选项进行解压时，会导致 exe/doc/xls 等类型的文件被解压。&lt;/p&gt;&lt;p&gt;2. 判断文件类型的方法为读取文件头，可能存在部分文件类型被误判/无法判断的情况。&lt;/p&gt;&lt;p&gt;3. 对于伪装为媒体格式的压缩文件，可以启用该选项进行尝试（部分文件可以正常进行测试/解压）。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>尝试处理未知格式文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_read_password_from_filename">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;从文件名中读取可能存在的密码。&lt;/p&gt;&lt;p&gt;支持的格式（以xxx指代密码）：&lt;/p&gt;&lt;p&gt;1. 以空格为间隔的两端字符：xxx 文件名/文件名 xxx&lt;/p&gt;&lt;p&gt;2. 以#开头，空格结尾的字符：#xxx 文件名&lt;/p&gt;&lt;p&gt;3. 以@开头，空格结尾的字符：@xxx 文件名&lt;/p&gt;&lt;p&gt;4. 在【】、[]、()中的字符：【xxx】文件名&lt;/p&gt;&lt;p&gt;5. 以&amp;quot;密码&amp;quot;、&amp;quot;解压码&amp;quot;、&amp;quot;解压密码&amp;quot;、&amp;quot;pw&amp;quot;、&amp;quot;PW&amp;quot;以及上述字符后带&amp;quot;:&amp;quot;、&amp;quot;：&amp;quot;开头，空格结尾的字符：密码xxx 文件名/解压密码:xxx 文件名&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>尝试从文件名中读取密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_test" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_write_filename">
            <property name="toolTip">
             <string>如果检索到正确的密码，则将改密码写入文件名中</string>
            </property>
            <property name="text">
             <string>将密码写入文件名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>密码文本格式:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>密码左侧字符:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_left_word">
              <property name="toolTip">
               <string>写入文件名时添加在左侧的字符</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>密码右侧字符:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_right_word">
              <property name="toolTip">
               <string>写入文件名时添加在右侧的字符</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>密码位置:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_pw_position">
              <property name="toolTip">
               <string>写入的位置，写入在最左端或者最右端</string>
              </property>
              <item>
               <property name="text">
                <string>最左端</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>最右端</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>文件名预览:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_preview_filename">
              <property name="text">
               <string>文件名示例</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_extract" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton_mode2_smart_extract">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;自动判断文件层级结构进行解压，类似于Bandizip。&lt;/p&gt;&lt;p&gt;如果仅存在单个文件/文件夹，则直接解压，如果存在多个文件且无上级文件夹，则自动创建文件夹并解压到该文件夹下。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>智能解压</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_mode2_direct_extract">
            <property name="toolTip">
             <string>不对文件层级结构进行检查，不进行处理直接解压</string>
            </property>
            <property name="text">
             <string>直接解压</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_mode2_extract_same_folder">
            <property name="toolTip">
             <string>解压到以压缩文件名命名的文件夹中</string>
            </property>
            <property name="text">
             <string>解压到同名目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_delete_origin">
            <property name="toolTip">
             <string>成功解压后删除源文件</string>
            </property>
            <property name="text">
             <string>解压后删除原文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_recursive_extract">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;解压嵌套的压缩文件，直到解压结果中不存在压缩文件。&lt;/p&gt;&lt;p&gt;实现方法：对解压的结果进行再一次的解压。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>递归解压嵌套压缩包</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>覆盖模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_cover_file">
              <property name="focusPolicy">
               <enum>Qt::FocusPolicy::WheelFocus</enum>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
              </property>
              <property name="toolTip">
               <string>解压文件存在冲突时的处理方法</string>
              </property>
              <property name="duplicatesEnabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>覆盖重复文件</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>跳过重复文件</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>重命名新文件</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>重命名旧文件</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_break_folder">
              <property name="text">
               <string>解散文件夹</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_break_folder">
              <property name="focusPolicy">
               <enum>Qt::FocusPolicy::WheelFocus</enum>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;处理多层级的文件夹结构，用于清理空文件夹。&lt;/p&gt;&lt;p&gt;移动到顶层目录：移动最底层的首个非空文件夹下的文件到顶层目录之下（保持文件层级结构，并删除空的该底层文件夹）。&lt;/p&gt;&lt;p&gt;移动底层文件夹：移动最底层的首个非空文件夹到顶层目录之外（并删除空的顶层目录）。&lt;/p&gt;&lt;p&gt;仅移动文件：移动所有文件到顶层目录之下（并删除空的子文件夹）。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <item>
               <property name="text">
                <string>移动到顶层目录</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>移动底层文件夹</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>仅移动文件</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_extract_output_folder">
            <property name="toolTip">
             <string>解压到该指定的文件夹，而不是源文件的同级文件夹中</string>
            </property>
            <property name="text">
             <string>解压至指定目录</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEdit_extract_output_folder"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_choose">
              <property name="text">
               <string>c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_open">
              <property name="text">
               <string>o</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_extract_filter">
            <property name="toolTip">
             <string>解压时过滤符合条件的文件（不解压该部分文件）</string>
            </property>
            <property name="text">
             <string>文件过滤规则</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_extract_filter_rule"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_top_window">
         <property name="toolTip">
          <string>保持窗口在前台</string>
         </property>
         <property name="text">
          <string>置顶程序窗口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_lock_size">
         <property name="toolTip">
          <string>锁定窗口尺寸，禁止修改</string>
         </property>
         <property name="text">
          <string>锁定窗口大小</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
